CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.25.3-dev
+* Bug - Fixed a bug where agent no longer redirected matadata query with multi-slash url to correct url [#1839](https://github.com/aws/amazon-ecs-agent/issues/1839) [#1844](https://github.com/aws/amazon-ecs-agent/pull/1844)
```

Comment:
we generally don't link issues in changelog.
also nit: `metadata`

Reply:
updated